src/page-multi-facility/FacilityRowRtValuePill.tsx

```diff
@@ -0,0 +1,62 @@
+import React from "react";
+import styled from "styled-components";
+
+import { RateOfSpreadType } from "../constants/EpidemicModel";
+import Colors from "../design-system/Colors";
+import PillCircle from "../design-system/PillCircle";
+
+const pillColors: {
+  [x: string]: {
+    [x: string]: string;
+  };
+} = {
+  [RateOfSpreadType.CONTROLLED]: {
+    text: Colors.green,
+    border: Colors.teal,
+  },
+  [RateOfSpreadType.INFECTIOUS]: {
+    text: Colors.darkRed,
+    border: Colors.orange,
+  },
+  [RateOfSpreadType.MISSING]: {
+    text: Colors.forest,
+    border: Colors.forest30,
+  },
+};
+
+const RtValuePill = styled.div<{ spreadType: string }>`
+  .pill-circle {
```

Comment:
can do `${PillCircle} { ... }` instead of having to add this classname

Reply:
I knew something like that worked. Ill update the others too.

Reply:
That didnt work...

Reply:
well that's mysterious ... https://styled-components.com/docs/advanced#referring-to-other-components is my source, possibly something is interfering with this?

Reply:
Not sure what the issue was but its working now. Thanks!